### ISSUE TITLE
提示不支持tensorflow-1.14.0-cp37-none-linux_armv7l.whl

### ISSUE THREAD

**Issue by wujianzhang1237** [raspberrypi/Raspberry-Pi-OS-64bit#42]
Is there a version of TensorFlow that can be installed? The package tensorFlow-1.14.0-CP37-none-linux_armv7l.WHL is used. It indicates that the platform is not supported

**Comment by lurch** [comment]
A .whl file is a python wheel, in which case you need to read https://blog.piwheels.org/raspberry-pi-os-64-bit-aarch64/ and I think this issue should be closed?
ping @bennuttall in case he can provide any additional info.

**Comment by bennuttall** [comment]
Correct. I've created issue #46 which you might want to pin, or point people at.

In the case of Tensorflow, it might be possible for the TF team to produce aarch64 wheels and distribute themselves. I'll ask.
